bin/i18n/sync-codeorg-in.rb

```diff
@@ -17,34 +17,11 @@ def sync_in
   localize_level_content
   localize_block_content
   run_bash_script "bin/i18n-codeorg/in.sh"
-  localize_pegasus_markdown_content
   # disable redaction of level content until the switch to remark is complete
   #redact_level_content
   redact_block_content
 end
 
-def localize_pegasus_markdown_content
-  # The in script grabs all the serialized pegasus strings, but we also want to
-  # localize some markdown pages. As of September 2018, there is exactly one
-  # page we want to localize, but we expect there to be more eventually.
-  markdown_to_localize = %w(
```

Comment:
@Hamms Why would we want Pegasus markdown content to be separate?  Thanks

Reply:
It needs to be in a separate project within crowdin, because the "skip export untranslated strings" option is incompatible with markdown content; we want that option enabled for our regular project, but disabled for markdown